va/va.h

```diff
@@ -539,9 +651,25 @@ typedef struct _VAConfigAttrib {
 #define VA_RC_CQP                       0x00000010
 /** \brief Variable bitrate with peak rate higher than average bitrate. */
 #define VA_RC_VBR_CONSTRAINED           0x00000020
+/** \brief Intelligent Constant Quality. Provided an initial ICQ_quality_factor,
+ *  adjusts QP at a frame and MB level based on motion to improve subjective quality. */
+#define VA_RC_ICQ			0x00000040
 /** \brief Macroblock based rate control.  Per MB control is decided
  *  internally in the encoder. It may be combined with other RC modes, except CQP. */
 #define VA_RC_MB                        0x00000080
+/** \brief Constant Frame Size */
```

Comment:
1. yes. 2. it will not take effect for CQP mode. 
we are discussing two features
1. MBBRC feature ---- it can only be enabled with BRC mode, CQP is not BRC mode
2. perMB Qp setting ---- only for CQP mode without BRC

per-mb Qp